guides/common/modules/proc_mqtt-tuning.adoc

```diff
@@ -0,0 +1,28 @@
+[id="REX_pull_provider_{context}"]
+= REX pull provider 
+
+Remote Execution has been around for a while now and has served SSH-based and Ansible execution very well. In many environments these are not options due to either security or networking concerns.
+
+Katello-agent has provided an alternative for a long time that worked in an environment where SSH was not available or the Foreman/Katello server or smart proxies did not have the ability to connect to a client. However katello-agent only supported a small set of actions around updating packages. There has been a long desire to create a ‘pull’ based provider that has the features of REX but with a deployment model similar tokatello-agent. 
+
+The solution uses MQTT as its messaging protocol and includes an MQTT client running on each Host. In orderto tune the MQTT, you can follow the following procedure on the Satellite or capsule server:
+
+.*Procedure*
+. Enable the feature using: `satellite-installer --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt` (this will change the ReX transport mode for the Satellite or capsule from the default SSH one to MQTT). It’s important to mention that one Satellite/capsule server can only use one transport mode: SSH or MQTT
```

Comment:
```suggestion
. Enable the feature using: `satellite-installer --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt` (this will change the ReX transport mode for the Satellite or capsule from the default SSH one to MQTT). It’s important to mention that one Satellite or Capsule server can only use one transport mode: SSH or MQTT
```

Reply:
Made a change. Thank you.